docs/output.md

```diff
@@ -6,548 +6,565 @@ This document describes the output produced by the pipeline. Most of the plots a
-- `[TUMORSAMPLE_VS_NORMALSAMPLE].cnvs.txt`
+- `<tumorsample_vs_normalsample>.cnvs.txt`
   - File with information about CNVS
-- `[TUMORSAMPLE_VS_NORMALSAMPLE].purityploidy.txt`
+- `<tumorsample_vs_normalsample>.purityploidy.txt`
   - File with information about purity and ploidy
-- `[TUMORSAMPLE_VS_NORMALSAMPLE].segments.txt`
+- `<tumorsample_vs_normalsample>.segments.txt`
   - File with information about copy number segments
-- `[TUMORSAMPLE_VS_NORMALSAMPLE].tumour_tumourBAF.txt` and `[TUMORSAMPLE_VS_NORMALSAMPLE].tumour_normalBAF.txt`
+- `<tumorsample_vs_normalsample>.tumour_tumourBAF.txt` and `<tumorsample_vs_normalsample>.tumour_normalBAF.txt`
   - file with beta allele frequencies
-- `[TUMORSAMPLE_VS_NORMALSAMPLE].tumour_tumourLogR.txt` and `[TUMORSAMPLE_VS_NORMALSAMPLE].tumour_normalLogR.txt`
+- `<tumorsample_vs_normalsample>.tumour_tumourLogR.txt` and `<tumorsample_vs_normalsample>.tumour_normalLogR.txt`
   - File with total copy number on a logarithmic scale
 
-The text file `[TUMORSAMPLE_VS_NORMALSAMPLE].cnvs.txt` contains predictions about copy number state for all the segments.
+The text file `<tumorsample_vs_normalsample>.cnvs.txt` contains predictions about copy number state for all the segments.
 The output is a tab delimited text file with the following columns:
 
 - _chr_: chromosome number
 - _startpos_: start position of the segment
 - _endpos_: end position of the segment
-- _nMajor_: number of copies of one of the allels (for example the chromosome inherited from the father)
-- _nMinor_: number of copies of the other allele (for example the chromosome inherited of the mother)
-
-The file `[TUMORSAMPLE].cnvs.txt` contains all segments predicted by ASCAT, both those with normal copy number (nMinor = 1 and nMajor =1) and those corresponding to copy number aberrations.
-
-For further reading and documentation see the [ASCAT manual](https://www.crick.ac.uk/research/labs/peter-van-loo/software).
+- _nMajor_: number of copies of one of the allels (for example the chromosome inherited of one parent)
+- _nMinor_: number of copies of the other allele (for example the chromosome inherited of the other parent)
+
+The file `<tumorsample_vs_normalsample>.cnvs.txt` contains all segments predicted by ASCAT, both those with normal copy number (nMinor = 1 and nMajor =1) and those corresponding to copy number aberrations.
+
+#### CNVKit
+
+[CNVKit](https://cnvkit.readthedocs.io/en/stable/) is a toolkit to infer and visualize copy number from high-throughput DNA sequencing data. It is designed for use with hybrid capture, including both whole-exome and custom target panels, and short-read sequencing platforms such as Illumina. For further reading and documentation, see the [CNVKit Documentation](https://cnvkit.readthedocs.io/en/stable/plots.html)
+
+_For normal or tumor-only samples_:
+
+**Output directory: `{outdir}/variantcalling/<sample>/cnvkit`**
+
+- `<sample>.antitargetcoverage.cnn`
```

Comment:
Just noticed that we're mixing `[sample]` and `<sample>` in the docs. I think we should be coherent throughout the docs